web/src/components/trace/ObservationPreview.tsx

```diff
@@ -33,6 +28,12 @@ export const ObservationPreview = (props: {
   currentObservationId: string;
   traceId: string;
 }) => {
+  const [selectedTab, setSelectedTab] = useState("preview");
+
+  useEffect(() => {
```

Comment:
if this is meant to reset the view between different observations, I think adding a `key` on the ObservationPreview would do the trick as well